### PR TITLE
added simple example using flask

### PR DESCRIPTION
**Pull request by penguindustin**
# Added

- example on how to use folium with flask

### CODE REVIEW

File: examples/flask_example.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask, render_template
```
[the remark this answers is not in the document]
Comment (penguindustin) [reply examples/flask_example.py]: fixed

**Comment by ocefpaf** [comment]
Awesome! Thanks @penguindustin!! This question pops up so frequently that it is a great addition to our docs. I'll leave it here for a while so @Conengmo can take a quick look at but I'm planning to merge it during the rest of the sprints.

**Comment by penguindustin** [comment]
@ocefpaf thanks for the mentoring! I enjoyed working on the project 😊

File: docs/index.rst [hunk 2]
```diff
@@ -32,6 +32,7 @@ Contents
    :maxdepth: 3
 
    installing
+   flask
```
Comment (Conengmo) [inline docs/index.rst]: If we are to include `flask.rst` in the documentation, I would suggest to move it below `quickstart.ipynb`. That file contains some more basic examples.

File: examples/flask_example.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```
Comment (Conengmo) [inline examples/flask_example.py]: Maybe add a docstring at the top of the file explaining how to use it. It's very basic but might be useful to people. Something about running this and then going to `http://127.0.0.1:5000/` in a browser.

File: docs/index.rst (hunk 2 above)
Comment (penguindustin) [reply docs/index.rst]: Whoops, I meant to put it under `quickstart` thanks for the catch

File: examples/flask_example.py (hunk 3 above)
Comment (penguindustin) [reply examples/flask_example.py]: Can do!

Comment (penguindustin) [reply examples/flask_example.py]: Added :D

**Comment by penguindustin** [comment]
@Conengmo I reordered the index and added some extra info to the example file. Should I replicate it in the `.rst`? I think there's a way to reference the actual python file instead of copy pasta into the `.rst` but I'm no expert in Sphinx docs.

**Comment by ocefpaf** [comment]
@Conengmo I think it is nice to have it in the docs and the best course of action, to avoid the duplication of code and changes of it going stale, is to reference the example file in the `rst` as @penguindustin suggests. I'm looking into how to do that.

**Comment by ocefpaf** [comment]
Thanks @penguindustin!

**Pull request merged**